build.sbt

```diff
@@ -3575,6 +3576,31 @@ lazy val `std-aws` = project
       "software.amazon.awssdk" % "sso"                     % awsJavaSdkV2Version,
       "software.amazon.awssdk" % "ssooidc"                 % awsJavaSdkV2Version
     ),
+    modulePath := {
+      val externalModIds = Seq(
+        "org.netbeans.api"       % "org-openide-util-lookup" % netbeansApiVersion,
+        "com.amazon.redshift"    % "redshift-jdbc42"         % redshiftVersion,
+        "software.amazon.awssdk" % "auth"                    % awsJavaSdkV2Version,
+        "software.amazon.awssdk" % "s3"                      % awsJavaSdkV2Version,
+        "software.amazon.awssdk" % "sso"                     % awsJavaSdkV2Version,
+        "software.amazon.awssdk" % "ssooidc"                 % awsJavaSdkV2Version,
+        "software.amazon.awssdk" % "http-client-spi"         % awsJavaSdkV2Version,
+        "software.amazon.awssdk" % "regions"                 % awsJavaSdkV2Version,
+        "software.amazon.awssdk" % "sdk-core"                % awsJavaSdkV2Version,
+        "software.amazon.awssdk" % "profiles"                % awsJavaSdkV2Version,
+        "software.amazon.awssdk" % "aws-core"                % awsJavaSdkV2Version
+      )
+      val externalMods = JPMSUtils.filterModulesFromUpdate(
+        (Compile / update).value,
+        externalModIds,
+        streams.value.log,
+        shouldContainAll = true
+      )
+      val ourMods =
+        (`std-base` / Compile / exportedProducts).value.map(_.data) ++
+        (`std-database` / Compile / exportedProducts).value.map(_.data)
```

Comment:
@Akirathan, cannot the `JPMSPlugin` iterate all project we `dependsOn` and extract modules from them automatically?

Reply:
This is a current technical limitation. Improvement tracked in https://github.com/enso-org/enso/issues/10726.

Reply:
It is really simpler: cfdda7b